### PR TITLE
Initial version of CassandraArchive

### PR DESCRIPTION
**Pull request by aliaksandr-litvinouski-epam**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2013 Alexander Litvinovsky
```
Comment (haruska) [inline LICENSE.txt]: Assign your copyright to Backupify.

**Comment by haruska** [comment]
This is good stuff. A little light on the tests.

File: LICENSE.txt (hunk 1 above)
Comment (aliaksandr-litvinouski-epam) [inline LICENSE.txt]: Yeah, will do! Looks like copyright has been generated by bundler, I generated gem template using it. Let me know if you want to change type of license

**Comment by aliaksandr-litvinouski-epam** [comment]
I am going to add more today.

**Comment by aliaksandr-litvinouski-epam** [comment]
I added more tests, please take a look

**Comment by haruska** [comment]
Looks good.

**Pull request merged**